js/hitbtc2.js

```diff
@@ -49,7 +49,9 @@ module.exports = class hitbtc2 extends hitbtc {
             },
             'urls': {
                 'logo': 'https://user-images.githubusercontent.com/1294454/27766555-8eaec20e-5edc-11e7-9c5b-6dc69fc42f5e.jpg',
-                'api': 'https://api.hitbtc.com',
+                'api': {
+                    'public': 'https://api.hitbtc.com',
```

Comment:
Hey ) Thx! What about `private` urls, tho? )

Reply:
Since there is only one end-point , wasn't sure it was needed ? 
But for consistency it is probably better , pushing it right now

Reply:
The `public` and `private` slug will be substituted in `this.urls['api'][api]`, and it will crash on private calls if there's no `this.urls['api']['private']`.